src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -108,7 +108,22 @@ protected Position handlePosition(Position position) {
                                 position.getAttributes().put(attribute.getAttribute(), (Boolean) result);
                                 break;
                             default:
-                                position.getAttributes().put(attribute.getAttribute(), result.toString());
+                                String attributeValue = null;
+                                String attributeName = attribute.getAttribute();
+                                if (attributeName.equals("alarm")
+                                        || attributeName.equals("event")) {
+                                    Object temp = position.getAttributes().
```

Comment:
There should be no need for temporary variable and conversion to string for alarm. I suggest we only do this for alarms.